src/scraping-defaults.js

```diff
@@ -28,7 +30,18 @@ function getCiphersBasedOnNode() {
     if (nodeVersion < 12) {
         return;
     }
-    return 'TLS_AES_256_GCM_SHA384:TLS_AES_128_GCM_SHA256:TLS_CHACHA20_POLY1305_SHA256';
+    return ensureModernTlsFirst();
+}
+
+/**
+ * @returns {string} ciphers list
+ */
+function ensureModernTlsFirst() {
+    const modernTlsCiphers = 'TLS_AES_256_GCM_SHA384:TLS_AES_128_GCM_SHA256:TLS_CHACHA20_POLY1305_SHA256:';
+    let defaultCiphers = crypto.constants.defaultCipherList.replace(modernTlsCiphers, '');
+    defaultCiphers = defaultCiphers.split(':').filter((cipher) => !!cipher);
+
+    return modernTlsCiphers + defaultCiphers.join(':');
```

Comment:
This is not very resilient. It would be better to deduplicate the ciphers with a `Set` instead of trying to look up the whole string. If the order of the ciphers in the `defaultCipherList` changes, the `replace` will stop working. There's also an extra `:` at the end of the `modernTlsCiphers`, but looking at the code I guess that's intentional. The whole thing is not very intuitive. A comment explaining why we need this would be nice.